x/nftmint/types/genesis.go

```diff
@@ -19,6 +19,12 @@ func (gs GenesisState) Validate() error {
 		return err
 	}
 
+	for _, classAttributes := range gs.ClassAttributesList {
+		if err := ValidateClassAttributes(*classAttributes, gs.Params); err != nil {
+			return err
+		}
+	}
```

Comment:
I think more validations will need to be added here for ClassNameIdLists and OwningClassIdLists.
As well as usage of usage of class ids on OwningClassIdLists that's not on ClassNameIdLists

Reply:
i know.
was thinking about whether i should add that kind of validation for those.
But, eventually i didn't do that since x/nftmarket doesn't have those type of validation, too, right?
do you even think i should add here now?

Reply:
Not a mandatory one but would be good to add for completeness.

Reply:
@mkXultra 
what do you think?
should i do this for this PR or do i just put this as an issue and give someone else this task?

Reply:
@taiki1frsh 
How long will it take?

Reply:
@mkXultra 
without test, it doesn't take an hour.
with out, i guess  around three or four hour.